app/src/pages/datasets/DatasetsTable.tsx

```diff
@@ -130,6 +135,60 @@ export function DatasetsTable(props: DatasetsTableProps) {
           return <Link to={to}>{row.original.name}</Link>;
         },
       },
+      {
+        header: "labels",
+        accessorKey: "labels",
+        enableSorting: false,
+        cell: ({ row }) => {
+          return (
+            <ul
+              css={css`
+                display: flex;
+                flex-direction: row;
+                gap: var(--ac-global-dimension-size-100);
+                min-width: 0;
+                flex-wrap: wrap;
+              `}
+            >
+              {row.original.labels.map((label) => (
+                <li
+                  key={label.id}
+                  css={css`
+                    min-width: 0;
+                    max-width: 200px;
+                  `}
+                >
+                  <Token
+                    color={label.color}
+                    css={css`
+                      min-width: 0;
+                      max-width: 100%;
+                      span {
+                        overflow: hidden;
+                        text-overflow: ellipsis;
+                        white-space: nowrap;
+                        min-width: 0;
+                      }
+                    `}
+                  >
+                    <span
+                      css={css`
+                        overflow: hidden;
+                        text-overflow: ellipsis;
+                        white-space: nowrap;
+                        min-width: 0;
+                      `}
+                      title={label.name}
+                    >
+                      {label.name}
+                    </span>
+                  </Token>
+                </li>
```

Comment:
You may be able to just use the \<Truncate> component for this

```suggestion
<li key={label.id}>
  <Token color={label.color}>
    <Truncate maxWidth={200}>
      {label.name}
    </Truncate>
  </Token>
</li>
```